apps/mull-api/src/app/user/user.resolver.ts

```diff
@@ -46,17 +46,22 @@ export class UserResolver {
     @Args('newAvatar', { type: /* istanbul ignore next */ () => GraphQLUpload, nullable: true })
     newAvatar: FileUpload
   ) {
-    var user = await this.userService.getUser(userInput.id);
+    const user = await this.userService.getUser(userInput.id);
     if (newAvatar) {
-      var media = user.avatar
-        ? await this.mediaService.updateFile(newAvatar, user.avatar)
-        : await this.mediaService.uploadFile(newAvatar);
+      if (user.avatar) {
+        var oldMediaId = user.avatar.id;
+        var media = await this.mediaService.updateFile(newAvatar, user.avatar);
+      } else {
+        var media = await this.mediaService.uploadFile(newAvatar);
+      }
       userInput = {
         ...userInput,
         avatar: media,
       };
     }
-    return await this.userService.updateUser(userInput);
+    const updatedUser = await this.userService.updateUser(userInput);
+    newAvatar && !!user.avatar ? await this.mediaService.deleteMedia(oldMediaId) : null;
```

Comment:
```suggestion
    newAvatar && !!user.avatar && this.mediaService.deleteMedia(oldMediaId);
```
I think this should be an equivalent expression.
Is this what you mean? 
```typescript
if (newAvatar && !user.avatar) this.mediaService.deleteMedia(oldMediaId);
```
Also, you could remove the await since the user doesn't need to wait on the old file to be deleted if the new one is stored and ready to go.